libs/service-portal/activities/src/screens/Sessions/Sessions.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAuth } from '@island.is/auth/react'
+import { Box, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { IntroHeader } from '@island.is/portals/core'
+
+import { m } from '../../lib/messages'
+
+const Sessions = () => {
+  const { formatMessage } = useLocale()
+  const { userInfo } = useAuth()
+
+  return (
+    <>
+      <IntroHeader
+        title={formatMessage(m.sessions)}
+        intro={formatMessage(m.sessionsHeaderIntro)}
+      />
+      <Box>
+        <Text>{ userInfo?.profile?.actor?.name ?? userInfo?.profile?.name}</Text>
```

Comment:
Displays the current name and nothing else as this stands now?

Reply:
Yes, wanted to merge this just as the shell. Then @GunnlaugurG is working on adding the table. We can change this to draft and merge it together. My thought was just to merge this as a shell to have smaller followup PRs.